src/NuGet.Core/NuGet.Commands/MSBuildPackTargetArgs.cs

```diff
@@ -32,4 +33,18 @@ public struct ProjectToProjectReference
         public string AssemblyName { get; set; }
         public string TargetPath { get; set; }
     }
+
+    public struct ContentMetadata
+    {
+        public string Target { get; set; }
+        public string Source { get; set; }
+        public string BuildAction { get; set; }
+        public string CopyToOutput { get; set; }
+        public string Flatten { get; set; }
+
+        public bool IsContentFile()
```

Comment:
Can we keep this method as a helper in the `MSBuildProjectFactory` class? It looks like it's only used there.